apps/recipes-react-components/package.json

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 6 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27100&commitId=ee7213bc69d8543afd1a5917698a1c79ceec411b&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1254 screenshots unchanged, 0 screenshots with different dimensions and 6 screenshots with visible difference.


  <details><summary><b>unknown</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|List.List Example Navigable Teams High Contrast.Focuses item.chromium.png|17865| Changed|
|List.List Example Navigable Teams High Contrast.chromium.png|18748| Changed|
|List.List Example Navigable Teams.Focuses item.chromium.png|117| Changed|
|List.List Example Navigable Teams.chromium.png|117| Changed|
|List.List Example Navigable.Focuses item.chromium.png|1363| Changed|
|List.List Example Navigable.chromium.png|1428| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>